edb/schema/objects.py

```diff
@@ -2688,11 +2703,15 @@ def get(
         name: Key_T,
         default: Optional[Object_T | NoDefaultT] = NoDefault,
     ) -> Optional[Object_T]:
-        items = dict(self.items(schema))
+        # TODO: Should we store an actual dict?
```

Comment:
Somewhere, we are probably iterating over names and calling `x.get(name)`, so this would be quadratic.

If I understand correctly, this `ObjectIndexBase` is used for example, to define pointers of an object or types in a module. Number of keys can be >50, right?

So I'd say yes, this should be dict.